pecos/core/base.py

```diff
@@ -148,6 +148,28 @@ def __init__(self, base_vect_param_list, norm_p):
         self.norm_p = c_int32(norm_p)
 
 
+class ClusteringParam(ctypes.Structure):
+    """
+    python class for handling struct ClusteringParam in clustering.hpp
+    """
+
+    _fields_ = [
+        ("partition_algo", c_int),
```

Comment:
`partition_algo`, `depth` and `kmeans_max_iter` use to be unsigned int. Better to be consistent.